apps/daimo-web/src/app/profile/[addr]/route.tsx

```diff
@@ -0,0 +1,33 @@
+import { NextResponse } from "next/server";
+import sharp from "sharp";
+
+import { rpc } from "../../../utils/rpc";
+
+type Context = {
+  params: { addr: string };
+};
+
+export async function GET(_: Request, { params }: Context) {
+  try {
+    const res = await rpc.getEthereumAccount.query({ addr: params.addr });
+    const profilePicture = res.linkedAccounts?.[0]?.pfpUrl;
+
+    if (profilePicture) {
+      const data = await fetch(profilePicture);
+      const blob = await data.blob();
+      const arrayBuffer = await blob.arrayBuffer();
+      const buffer = Buffer.from(arrayBuffer);
+
+      const result = await sharp(buffer).resize(64, 64).toBuffer();
+      const resultBlob = new Blob([result], { type: "image/jpeg" });
+      const response = new NextResponse(resultBlob);
```

Comment:
looks great

we should cache (addr > Blob | null) , either in memory lru cache or db table

& either here, or in a followup pr

Reply:
Next.js apparently caches the results of `fetch`-es like this one. But I agree that using a LRU cache is more explicit and gives more control over what is cached, where and for how long.